nifi-extension-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-hikari-dbcp-service/src/main/java/org/apache/nifi/dbcp/HikariCPConnectionPool.java

```diff
@@ -213,6 +217,8 @@ public class HikariCPConnectionPool extends AbstractControllerService implements
 
     private volatile HikariDataSource dataSource;
     private volatile KerberosUser kerberosUser;
+    // Hold an instance of the driver so we can properly de-register it.
+    private volatile Driver registeredDriver;
```

Comment:
Is it necessary to keep this instance, or could `DriverManager.getDrivers()` be used to get registered drivers and compare against the configured class name?